### PR TITLE
Redirect run() input from /dev/null, adjust formatting

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

**Review by DaanDeMeyer — CHANGES_REQUESTED** [review]
Looks great, but let's not do the extra whitespace after [ and before ]

File: mkosi/__init__.py [hunk 1]
```diff
@@ -483,18 +483,11 @@ def install_boot_loader(state: MkosiState) -> None:
...
-                        "--key",
-                        state.config.secure_boot_key,
-                        "--cert",
-                        state.config.secure_boot_certificate,
-                        "--output",
-                        f"{f}.signed",
-                        f,
-                    ],
-                )
+                run([ "sbsign",
```
[the remark this answers is not in the document]
Comment (behrmann) [reply mkosi/__init__.py:486]: I see the value in having option and value on the same line, that's definitely an improvement over the black style, but having them indented separately as currently is visually more distinct, also it makes diffs smaller and these things do tend to change.

[the remark this answers is not in the document]
Comment (keszybz) [reply mkosi/__init__.py]: OK, updated.

**Comment by keszybz** [comment]
Hmm, unfortunately this is broken.

**Comment by keszybz** [comment]
The failure was:
```
‣ Saving manifest image.manifest
+ setfacl --physical --modify user:65436:rwx -
Traceback (most recent call last):
File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/zbyszek/src/mkosi/mkosi/__main__.py", line 62, in <module>
    main()
  File "/usr/lib64/python3.11/contextlib.py", line 81, in inner
    return func(*args, **kwds)
  File "/home/zbyszek/src/mkosi/mkosi/__main__.py", line 57, in main
    run_verb(load_args(args))
  File "/home/zbyszek/src/mkosi/mkosi/__init__.py", line 2431, in run_verb
    fork_and_wait(target)
  File "/home/zbyszek/src/mkosi/mkosi/run.py", line 166, in fork_and_wait
    result = target()
             ^^^^^^^^
  File "/home/zbyszek/src/mkosi/mkosi/__init__.py", line 2426, in target
    build_stuff(uid, gid, config)
  File "/home/zbyszek/src/mkosi/mkosi/__init__.py", line 1889, in build_stuff
    acl_toggle_remove(state.config, state.config.cache_dir, state.uid, allow=True)
  File "/home/zbyszek/src/mkosi/mkosi/__init__.py", line 861, in acl_toggle_remove
    ret = run(["setfacl",
          ^^^^^^^^^^^^^^^
  File "/home/zbyszek/src/mkosi/mkosi/run.py", line 224, in run
    return subprocess.run(cmdline,
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/subprocess.py", line 538, in run
    raise ValueError('stdin and input arguments may not both be used.')
ValueError: stdin and input arguments may not both be used.
```
Unfortunately the CI doesn't catch this. Most likely because we're running in a container and don't do ACL stuff.

**Review by DaanDeMeyer — APPROVED** [review]

**Pull request merged**